src/platforms/esp32/components/avm_sys/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
+# This file is part of AtomVM.
+#
+# Copyright 2022 Davide Bettio <davide@uninstall.it>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+#
+
+set(AVM_SYS_COMPONENT_SRCS
+    "sys.c"
+    "platform_nifs.c"
+    "platform_defaultatoms.c"
+)
+
+idf_component_register(SRCS ${AVM_SYS_COMPONENT_SRCS}
```

Comment:
Not a fig fan of this indentation style.
```
prefer(
    args,
    indented,
    on_col_4
)
```

Reply:
Fixed.